### PR TITLE
updates documentation for PathDirectives

### PR DESCRIPTION
**Pull request by Prithvirajbilla**

### CODE REVIEW

**Comment by johanandren** [comment]
OK TO TEST

**Comment by johanandren** [comment]
This failure looks related:

```
[error] /home/jenkins/workspace/workspace/pr-validator-akka-http/docs/src/test/java/docs/http/javadsl/server/directives/PathDirectivesExamplesTest.java:168:1: cannot find symbol
[error]   symbol:   method separateOnSlashes(java.lang.String)
[error]   location: class docs.http.javadsl.server.directives.PathDirectivesExamplesTest
[error]     path(separateOnSlashes("foo/bar"), () -> complete(StatusCodes.OK));
```

File: docs/src/test/scala/docs/http/scaladsl/server/directives/PathDirectivesExamplesSpec.scala [hunk 1]
```diff
@@ -19,13 +19,13 @@ class PathDirectivesExamplesSpec extends RoutingSpec {
   // matches /foo/
   path("foo"./)
 
-  // matches "/foo/bar"
```
Comment (Prithvirajbilla) [inline docs/src/test/scala/docs/http/scaladsl/server/directives/PathDirectivesExamplesSpec.scala]: good catch! Thanks for changing this. 

I am not very good  at recognizing these type of mistakes and I am working on it.

Comment (ktoso) [reply docs/src/test/scala/docs/http/scaladsl/server/directives/PathDirectivesExamplesSpec.scala]: No worries :) Thanks for the PR!

**Pull request merged**